string-manipulation-1/src/is-vowel.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable no-unused-vars */
+function isVowel(char) {
+  switch (char) {
+    case 'a':
+      return true;
+    case 'E':
+      return true;
+    case 'I':
+      return true;
+    case 'o':
+      return true;
+    case 'u':
+      return true;
+    default:
+      return false;
```

Comment:
this will fail if the string contains a lowercase `e`, try to generalize your solution to work with any input character.